src/pyoframe/model.py

```diff
@@ -90,3 +90,4 @@ def __repr__(self) -> str:
 
     to_file = to_file
     solve = solve
+    get_var_map = get_var_map
```

Comment:
I think this is unnecessary since you can do `model.var_map` and get the same result.

Reply:
But the function does more than just returning the var_map, in the case that this is None.

Reply:
Hmm what about accessing `model.io_mappers`. I guess I'm missing the context for how this will be used

Reply:
I think it would be good to avoid recreating variable maps when possible, hence why I'm not a fan of exposing `get_var_map`